### PR TITLE
test: add test for all-words-in-tree

### PR DESCRIPTION
**Pull request by ChristieRobson**
Added test, see issue https://github.com/knaxus/problem-solving-javascript/issues/144

### CODE REVIEW

**Comment by TheSTL** [comment]
Hi @ChristieRobson your commits are not displayed, setup local git properly and push again.

**Comment by ChristieRobson** [comment]
> Hi @ChristieRobson your commits are not displayed, setup local git properly and push again.

I'm confused what you mean by this, could you please elaborate? In the PR I can see one commit and 1 file showing as being changed.

**Comment by ashokdey** [comment]
Thanks for your contribution @ChristieRobson, however, the same rules apply here. Please go through my comment in #149

File: src/_DataStructures_/Trees/Trie/all-words-in-trie/all-words-in-trie.test.js [hunk 1]
```diff
@@ -13,4 +13,49 @@ describe('Data Structure : Trie : All Words In Tree', () => {
...
+    expect(() => {
+      allWordsInTrie([]);
+    }).toThrow('Cannot read property \'0\' of undefined');
+  });
+
+  it('passing an array not in a Trie will throw an error ', () => {
+    expect(() => {
+      allWordsInTrie(['bed', 'ball', 'apple']);
+    }).toThrow('Cannot read property \'0\' of undefined');
+  });
```
Comment (ashokdey) [inline src/_DataStructures_/Trees/Trie/all-words-in-trie/all-words-in-trie.test.js]: It's great to see you @ChristieRobson, you seem to be a quick learner. I appreciate your efforts and I will encourage you to pick more issues from the pool. 

For the above selected chunk of code, please follow the same instructions given in #149

Comment (ChristieRobson) [reply src/_DataStructures_/Trees/Trie/all-words-in-trie/all-words-in-trie.test.js]: Tests updated with specifc error

**Comment by ashokdey** [comment]
Good work @ChristieRobson 🎉 

Do star the repo if you liked working on it and also share it with the world. Thanks

**Pull request merged**